resources/texts/change_log.txt

```diff
@@ -1,3 +1,61 @@
+[4.13]
+Beta
+
+* Sync material profiles 
+With Ultimaker Cura 4.13 beta, we give you access to a seamless material experience for Ultimaker Material Alliance materials – with the ease of use you’ve come to expect from Ultimaker materials. You can easily synchronize your Material Alliance profiles with your S-line Ultimaker hardware, at the click of a button. 
+
+* New print profile
+A new print profile with 0.3mm layer height for PLA Tough PLA, PVA and BAM for Ultimaker S-line printers
+
+* 3MF thumbnail
+Show the model in the thumbnail of a .3mf file, contributed by fieldOfView
+
+* Infill density
+When printing with a 100% infill the infill pattern will change to ZigzZag for all Ultimaker print profiles
+
+* User login authentication
+We’ve streamlined the user login authentication by removing any restrictions, especially for strict enterprise-level IT requirements. 
+
+* Other new features and improvements:
+- TPU top layers 
+- Add warning icon to show which extruder is causing the configuration to be 'Not Supported', contributed by fieldOfView
+- Show what's new pages with every Cura build
+- Speed up loading of settings list
+- Re-use vertex buffer objects in rendering
+- Add Build Volume Temperature value to ChangeAtZ, contributed by legend069
+- Allow plugins to have multiple views, contributed by Tyronnosaurus
+
+* Bug fixes:
+- Reduced flow for 100% density parts
+- Fixed a bug in Surface Mode where small line-segments were created
+- Changed the Russian translation for 'nozzle', contributed by mlapkin
+- Fixed a visualization bug where layer lines were renderend in weird directions
+- Fixed a crash when recieving incomplete cloud API responses
+- Add SET_RPATH option to CMake, contributed by boomanaiden154
+- Fixed initial layer bed and print head temperature for Snapmaker profile, contributed by prueker
+- Fixed shader compilation on some GPUs, contributed by fieldOfView
+- Fixed a bug where Cross 3D infill pattern vertical angles varies wildly
+- Bridge Skin Density can be set above 100%
+- Fixed tiny travel moves when monotonic ordering was enabled
+- Fix crash when using 'Select face to align to the build plate', contributed by eliadevito
+- Fixed a bug in fuzzy skin where sometimes it produced weird long overshoots, contributed by BagelOrb
+- Fixed undo and redo for support blockers
+- Fixed a bug where the Native CAD plugin wouldn't loading
```

Comment:
@fvrmr grammar. It should be: `Fixed a bug where the Native CAD plugin didn't load`
I also think this description doesn't cover the bug. A more appropriate message is: `Fixed a bug when a backup restored plugin, which uses the Central File Storage (such as Native CAD plugin) resulted in a corrupt plugin.` Or something along that line.